scraping/x/adajobo_scrapper.py

```diff
@@ -0,0 +1,372 @@
+import asyncio
+import threading
+import traceback
+import bittensor as bt
+from typing import List
+from common import constants
+from common.data import DataEntity, DataLabel, DataSource
+from common.date_range import DateRange
+from scraping.scraper import ScrapeConfig, Scraper, ValidationResult
+from scraping.apify import ActorRunner, RunConfig
+from scraping.x.model import XContent
+from scraping.x import utils
+import datetime as dt
+
+
+class AdajoboTwitterScraper(Scraper):
+    """
+    Scrapes tweets using the Adajobo Twitter Scraper: https://console.apify.com/actors/heLL6fUofdPgRXZie.
+    """
+
+    ACTOR_ID = "61RPP7dywgiy0JPD0"
+
+    SCRAPE_TIMEOUT_SECS = 120
+
+    BASE_RUN_INPUT = {
+        "maxRequestRetries": 5,
+        "searchMode": "live",
+    }
+
+    # As of 2/5/24 this actor only takes 256 MB in the default config so we can run a full batch without hitting shared actor memory limits.
```

Comment:
Is this also true for this new actor?

Reply:
We don't need to specify searchMode for new actor, it will just ignored.